extension/public/playground/js/index.js

```diff
@@ -1,8 +1,24 @@
+import { getPublicKey, signTransaction, isConnected } from "@stellar/lyra-api";
```

Comment:
updating the playground to make sure it works - but this will be moving to docs in the next PR